src/main/java/megamind/main/Launcher.java

```diff
@@ -0,0 +1,13 @@
+package megamind.main;
+
+import javafx.application.Application;
+import megamind.gui.Main;
+
+/**
+ * Acts as the entry point for Cookie application
+ */
+public class Launcher {
```

Comment:
I like how you split up the different files into different packages so that it is easier to read!